docker/Dockerfile.arm

```
@@ -19,7 +33,12 @@ RUN vcpkgCommitId=$(grep '.builtin-baseline' vcpkg.json | awk -F: '{print $2}' |
 
 WORKDIR /opt/vcpkg
 COPY vcpkg.json /opt/vcpkg/
-RUN VCPKG_FORCE_SYSTEM_BINARIES=1 /opt/vcpkg/vcpkg --feature-flags=binarycaching,manifests,versions install
+RUN --mount=type=secret,id=github_token \
```

Hadolint (2.14.0)

[error] Expected test to end here (don't wrap commands in []/[[]]). Fix any mentioned problems and try again.

(SC1072)